.github/workflows/test.yml

```diff
@@ -8,22 +8,13 @@ on:
 
 jobs:
   test:
-    services:
-      presto:
-        image: ghcr.io/popsql/prestodb-sandbox:0.284
-        ports:
-          - "18080:8080"
-      trino:
-        image: trinodb/trino:418
-        ports:
-          - "18081:8080"
-
     runs-on: ubuntu-latest
     strategy:
       matrix:
         node-version: [ 16.x, 18.x, 20.x ]
     steps:
       - uses: actions/checkout@v3
+      - run: docker compose up --detach --wait
```

Comment:
This does the same thing as using the services key, where `--wait` gets the command to wait until the containers are marked as healthy before continuing, similar to what `services` was doing.

This simplifies the test setup to better match local development, and that there was no way to mount a volume from the local repository (as do nginx configuration) via the `services` setup.